specification/azurestackhci/resource-manager/Microsoft.AzureStackHCI/stable/2022-10-01/offers.json

```diff
@@ -0,0 +1,302 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2022-10-01",
+    "title": "AzureStackHCI",
+    "description": "Azure Stack HCI management service"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.AzureStackHCI/clusters/{clusterName}/publishers/{publisherName}/offers": {
+      "get": {
+        "operationId": "Offers_ListByPublisher",
+        "description": "List Offers available for a publisher within the HCI Cluster.",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/SubscriptionIdParameter"
```

Comment:
>v2 [](http://example.com/codeflow?start=69&length=2)

Recommend using v3 common types for new APIs. This comment applies throughout this file and other added code of the PR.